src/Language/PureScript/Errors.hs

```diff
@@ -701,9 +707,22 @@ prettyPrintSingleError (PPEOptions codeColor full level showDocs relPath) e = fl
       paras [ line $ "In the FFI module for " <> markCode (runModuleName mn) <> ":"
             , indent . paras $
                 [ line $ "The identifier " <> markCode ident <> " contains a prime (" <> markCode "'" <> ")."
-                , line "Primes in identifiers exported from FFI modules are deprecated and won’t be supported in the future."
+                , line "Primes are not allowed in identifiers exported from FFI modules."
```

Comment:
The error type is `DeprecatedFFIPrime`, but this error indicates they are no longer allowed at all. Should a new error be made for them? In other words, if I'm on v0.14.x, I should still be able to go to the doc repo and see the `DeprecatedFFIPrime` page. However, if I'm on v0.15.x in a different project, jumping to the corresponding page shouldn't tell me that these are deprecated. Rather, it should bring me to a separate page that says they are not allowed at all.

Reply:
Is there no advantage to having one page that says that this is deprecated on <0.15 and not allowed on >=0.15? It might add some weight to the warning for the v0.14-ers—‘X is deprecated’ is a less useful signal than ‘X is deprecated and won't be available in version Y’.

Reply:
Good point! I didn't think about this as a migration touch point.